docs/cunumeric/source/user/notebooks/stencil.ipynb

```diff
@@ -0,0 +1,221 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "G = make_grid(100)\n",
+    "\n",
+    "plt.pcolor(G);"
```

Comment:
semicolon

Reply:
That was intentional to suppress the annoying plot object string repr

Reply:
oh, huh, okay. Haven't seen that before. 👍